.env.example

```
@@ -1,7 +1,6 @@
 RPC_URL=${RPC_URL}
-PRIVATE_KEY=${PRIVATE_KEY}
 VAULT_ADDRESS=${VAULT_ADDRESS}
-BOT_WALLET=${BOT_WALLET}
+FIREBLOCKS_API_USER_ID=${FIREBLOCKS_API_USER_ID}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FIREBLOCKS_API_USER_ID key should go before the RPC_URL key

(UnorderedKey)